observability-kit-demo-hilla/frontend/index.ts

```diff
@@ -2,6 +2,9 @@
 import { Router } from '@vaadin/router';
 import { routes } from './routes.js';
 import { appName, location, setLocation } from './stores/location-store.js';
+import initTelemetry from 'Frontend/utils/telemetry.js';
+
+initTelemetry();
```

Comment:
This feels like something we should provide as a kit feature instead that of a utility function in demo code. I've created #218 to track this.